src/server/game/Handlers/NPCHandler.cpp

```diff
@@ -734,7 +734,7 @@ void WorldSession::HandleRepairItemOpcode(WorldPacket& recvData)
 
         Item* item = _player->GetItemByGuid(itemGUID);
         if (item)
-            _player->DurabilityRepair(item->GetPos(), true, discountMod, guildBank != 0);
```

Comment:
is guildBank ignored now in this case or ?

Reply:
> is guildBank ignored now in this case or ?

Yes. We can't repair one specific item with guild money, only for the player's own money. If specific item guid received we can ignore `guildBank` parameter